macos/README.md

```diff
@@ -7,8 +7,8 @@ For our CI hosts we consider the jenkins agent process to be the most critical t
 
 ## Preparing the Jenkins agent property list
 
-The agent plist was written when dosa was reimaged December 2017.
-It expects that the directory `$HOME/jenkins-agent` exists and that it contains the Jenkins agent program `slave.jar`.
+The agent plist was written when dosa was reimaged December 2017 and updated to use the Swarm client in Feb 2018.
+It expects that the directory `$HOME/jenkins-agent` exists and that it contains the Jenkins agent program `swarm-client-3.8.jar`.
```

Comment:
What exactly is the swarm client?  Where can I get it?  Some documentation and/or links would be helpful to someone like me who isn't following all of this stuff.

Reply:
f181e7b4bf8106704d94e72380f2ffe8fc6109fd adds a sentence about the swarm client jar to the macos docs and links to it in both docs. Happy to add more if you think it's valuable to do so. I've been enmired in this for a bit so I trust your judgement as to what explanations need improvement.

Reply:
Excellent, thanks.